certified-connectors/LnkBio/apiDefinition.swagger.json

```diff
@@ -0,0 +1,141 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "description": "Automate publishing workflow on [Lnk.Bio](https://lnk.bio)",
```

Comment:
@andreaolivato please define a proper description for your connector. You can include any external links in your Intro.md file later in your ISV Studio submission. Please follow our documents [here](https://learn.microsoft.com/en-us/connectors/custom-connectors/certification-submission#write-a-description-for-your-connector) and [here](https://learn.microsoft.com/en-us/connectors/custom-connectors/connector-string-guidance#connector-info)